homeassistant/components/litterrobot/vacuum.py

```diff
@@ -97,15 +103,43 @@ def status(self) -> str:
     async def async_turn_on(self, **kwargs: Any) -> None:
         """Turn the cleaner on, starting a clean cycle."""
         await self.robot.set_power_status(True)
+        if self._turn_on_call_reported:
+            return
+        self._turn_on_call_reported = True
+        ir.async_create_issue(
+            self.hass,
+            DOMAIN,
+            "service_deprecation_turn_on",
+            breaks_in_ha_version="2024.2.0",
+            is_fixable=False,
+            severity=ir.IssueSeverity.WARNING,
+            translation_key="service_deprecation_turn_on",
+        )
 
     async def async_turn_off(self, **kwargs: Any) -> None:
         """Turn the unit off, stopping any cleaning in progress as is."""
         await self.robot.set_power_status(False)
+        if self._turn_off_call_reported:
+            return
+        self._turn_off_call_reported = True
+        ir.async_create_issue(
+            self.hass,
+            DOMAIN,
+            "service_deprecation_turn_off",
+            breaks_in_ha_version="2024.2.0",
+            is_fixable=False,
+            severity=ir.IssueSeverity.WARNING,
+            translation_key="service_deprecation_turn_off",
+        )
 
     async def async_start(self) -> None:
         """Start a clean cycle."""
         await self.robot.start_cleaning()
```

Comment:
What happens if `self.robot.set_power_status(False)` has been called, will this still work, or do we need to call `self.robot.set_power_status(True)` here?

Reply:
`self.robot.set_power_status(True)` would need to be called first if the robot is off, otherwise nothing will happen.

Reply:
OK, I updated the code to call `self.robot.set_power_status(True)`